meerk40t/balormk/device.py

```diff
@@ -1772,26 +1759,24 @@ def ant_points(points, steps):
             "ants",
             help=_("Marching ants of the given element path."),
             input_type=(None, "elements"),
-            output_type="shapes",
+            output_type="geometry",
         )
         def element_ants(command, channel, _, data=None, quantization=50, **kwargs):
             """
             Draws an outline of the current shape.
             """
             if data is None:
                 data = list(self.elements.elems(emphasized=True))
-            points_list = []
-            points = list()
+            geom = Geomstr()
             for e in data:
                 try:
-                    path = e.as_path()
+                    path = e.as_geometry()
                 except AttributeError:
                     continue
-                for i in range(0, quantization + 1):
-                    x, y = path.point(i / float(quantization))
-                    points.append((x, y))
-                points_list.append(list(ant_points(points, int(quantization / 2))))
-            return "shapes", [Polygon(*p) for p in points_list]
+                ants = list(ant_points(path.as_interpolated_points(interpolate=quantization), int(quantization/2)))
```

Comment:
This is not 1:1 with the old code. We may not be interpolating parts of different lines whereas previously we may have interpolated long lines.